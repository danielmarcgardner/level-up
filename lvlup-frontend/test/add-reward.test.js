import AddReward from '../src/components/Admin/admin-add-reward/add-reward-main';
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

test('AddReward should render', () => {
  const add = shallow(
    <AddReward />,
  );
  expect(toJson(add)).toMatchSnapshot();
});
