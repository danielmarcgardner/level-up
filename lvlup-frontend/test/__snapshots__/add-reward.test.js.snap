exports[`test AddReward should render 1`] = `
<div
  className="add-reward">
  <Connect(ReduxForm) />
</div>
`;
